readme.md

```diff
@@ -78,25 +78,6 @@ try {
 }
 ```
 
-### Cancelling a spawned process
-
-```js
-import {execa} from 'execa';
-
-const subprocess = execa('node');
-
-setTimeout(() => {
-	subprocess.cancel();
-}, 1000);
-
-try {
-	await subprocess;
-} catch (error) {
-	console.log(subprocess.killed); // true
-	console.log(error.isCanceled); // true
```

Comment:
We might want to keep the following from this example:
  - Printing `isCanceled`
  - Showing that aborts make the process rejects its returned promise
  - Calling `abort()` inside `setTimeout()`, since this mimics how it would be used in real use cases (calling it right after the process has started is unlikely)